app/assets/stylesheets/application.sass.scss

```diff
@@ -3,4 +3,42 @@
 /* Backdrop is only displayed when dialog is opened with dialog.showModal() */
 dialog::backdrop {
   background: rgba(0, 0, 0, 0.5);
+}
+
+// Evaluation Form Step 2
+.eval-scale-heading {
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 25px;
+}
+
+.usa-fieldset {
+  margin-top: 20px;
+}
+
+.radio-group {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+}
+
+.radio-item {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+}
```

Comment:
I'm not sure what this does, but it doesn't seem to change anything when I remove it